packages/ringcentral-widgets/components/LogSection/index.js

```diff
@@ -124,9 +126,23 @@ export default class LogSection extends Component {
     if (result) {
       return this.genLogBasicInfo();
     }
+    function disabledToCallControl() {
```

Comment:
Seems can use the function `isRingingInboundCall` from `ringcentral-integration/lib/callLogHelpers` to repace this function